pkg/docker/client.js

```diff
@@ -726,6 +729,18 @@
             return connected.promise();
         };
 
+        this.containers_for_image = function containers_for_image(id) {
+            util.docker_debug('containers search on image id: ', id);
+            return http.get('/v1.12/containers/json', { all: 1 , filters: JSON.stringify({ ancestor: [ id ] }) })
+                .fail(function(ex) {
+                    console.log(ex);
```

Comment:
Do we really want to log and debug? If we don't expect this to happen, this should perhaps be a .warn()?

Reply:
I think it's a log I forgot to remove, woops